transcendence-app/src/chat/infrastructure/generic-chat.entity.ts

```diff
@@ -0,0 +1,43 @@
+export enum ChatType {
+  ONE_TO_ONE = 'oneToOne',
+  CHATROOM = 'chatroom',
+}
+
+export interface GenericChatData {
+  avatarId: string;
+  avatarX: number;
+  avatarY: number;
+  rtti: ChatType;
+  name: string;
+  id: string;
+  isPublic?: boolean;
+  lastMsgSenderUsername?: string;
+  lastMessage?: string;
+  lastMessageDate: Date;
+}
+
+export class GenericChat {
```

Comment:
praise: Great idea to add this generic chat type :+1:

Reply:
Thank you